app/models/filters/hmis_export.rb

```diff
@@ -9,7 +9,7 @@ class HmisExport < FilterBase
     include ArelHelper
     attribute :start_date, Date, default: 1.years.ago.to_date
     attribute :end_date, Date, default: Date.current
-    attribute :version, String, default: '2022'
+    attribute :version, String, default: '2024'
```

Comment:
Is this OK? tripped me up a couple times that it was still defaulting to 2022.

Reply:
Yes!  Thank you.